homeassistant/package_constraints.txt

```diff
@@ -65,7 +65,7 @@ uv==0.4.22
 voluptuous-openapi==0.0.5
 voluptuous-serialize==2.6.0
 voluptuous==0.15.2
-webrtc-models==0.0.0b2
+webrtc-models==0.1.0
```

Comment:
This should be a separate PR.

Reply:
Yep, it's for testing now. I was hoping someone will do it before me; if not, I will do it myself. Same with the `hass-nabucasa`.